[DOC] Clean-up of the Managing RBAC resources chapter (#9206)
strimzi/strimzi-kafka-operator: documentation/modules/operators/ref-operator-cluster-rbac-resources.adoc

|strimzi-kafka-broker
|Access rights to allow Kafka brokers to get the topology labels from Kubernetes worker nodes when rack-awareness is used.

|strimzi-entity-operator
|Access rights used by the Topic and User Operators to manage Kafka users and topics.

|strimzi-kafka-client
|Access rights to allow Kafka Connect, MirrorMaker (1 and 2), and Bridge to get the topology labels from Kubernetes worker nodes when rack-awareness is used.

|===

== `ClusterRoleBinding` resources

The Cluster Operator uses `ClusterRoleBinding` and `RoleBinding` resources to associate its `ClusterRole` with its `ServiceAccount`:
Cluster role bindings are required by cluster roles containing cluster-scoped resources.

.`ClusterRoleBinding` resources
[cols="1m,1",options="header"]
|===

| Name
| Description

|strimzi-cluster-operator
|Grants the Cluster Operator the rights from the `strimzi-cluster-operator-global` Cluster Role

|strimzi-cluster-operator-kafka-broker-delegation
|Grants the Cluster Operator the rights from the `strimzi-entity-operator` Cluster Role

|strimzi-cluster-operator-kafka-client-delegation

> |Grants the Cluster Operator the rights from the `strimzi-kafka-client` Cluster Role

Reviewer: ```suggestion
|Grants the Cluster Operator the rights from the `strimzi-kafka-client` cluster role.
```

|===

.`RoleBinding` resources
[cols="1m,1",options="header"]
|===

| Name
| Description

|strimzi-cluster-operator

> |Grants the Cluster Operator the rights from the `strimzi-cluster-operator-namespaced` Cluster Role

Reviewer: ```suggestion
|Grants the Cluster Operator the rights from the `strimzi-cluster-operator-namespaced` cluster role.
```

|strimzi-cluster-operator-leader-election

> |Grants the Cluster Operator the rights from the `strimzi-cluster-operator-leader-election` Cluster Role

Reviewer: ```suggestion
|Grants the Cluster Operator the rights from the `strimzi-cluster-operator-leader-election` cluster role.
```

|strimzi-cluster-operator-watched

> |Grants the Cluster Operator the rights from the `strimzi-cluster-operator-watched` Cluster Role

Reviewer: ```suggestion
|Grants the Cluster Operator the rights from the `strimzi-cluster-operator-watched` cluster role.
```

|strimzi-cluster-operator-entity-operator-delegation

> |Grants the Cluster Operator the rights from the `strimzi-cluster-operator-entity-operator-delegation` Cluster Role

Reviewer: ```suggestion
|Grants the Cluster Operator the rights from the `strimzi-cluster-operator-entity-operator-delegation` cluster role.
```

|===

== `ServiceAccount` resources

The Cluster Operator runs using the `strimzi-cluster-operator` `ServiceAccount`.

> This `ServiceAccount` is used to grant it the privileges it required to manage the operands.

Reviewer: ```suggestion
This service account grants it the privileges it requires to manage the operands.
```

> The Cluster Operator will create additional `ClusterRoleBinding` and `RoleBinding` resources to delegate some of these RBAC rights to the operands.

Reviewer: ```suggestion
The Cluster Operator creates additional `ClusterRoleBinding` and `RoleBinding` resources to delegate some of these RBAC rights to the operands.
```

> Each of the operands uses its own Service Account created by the Cluster Operator.

Reviewer: ```suggestion
Each of the operands uses its own service account created by the Cluster Operator.
```

> That allows it to follow the principle of least privilege and give the operands only the access rights that are really need.

Reviewer: ```suggestion
This allows the Cluster Operator to follow the principle of least privilege and give the operands only the access rights that are really need.
```

.`ServiceAccount` resources
[cols="1m,1",options="header"]
|===
| Name
| Used by

> |_<cluster_name>_-zookeeper

Reviewer: ```suggestion
|<cluster_name>-zookeeper
```

|ZooKeeper pods

> |_<cluster_name>_-kafka

Reviewer: ```suggestion
|<cluster_name>-kafka
```

|Kafka broker pods

> |_<cluster_name>_-entity-operator

Reviewer: ```suggestion
|<cluster_name>-entity-operator
```

|Entity Operator

> |_<cluster_name>_-cruise-control

Reviewer: ```suggestion
|<cluster_name>-cruise-control
```

|Cruise Control pods

> |_<cluster_name>_-kafka-exporter

Reviewer: ```suggestion
|<cluster_name>-kafka-exporter
```

|Kafka Exporter pods

> |_<cluster_name>_-connect

Reviewer: ```suggestion
|<cluster_name>-connect
```

|Kafka Connect pods

> |_<cluster_name>_-mirror-maker

Reviewer: ```suggestion
|<cluster_name>-mirror-maker
```

|MirrorMaker pods

> |_<cluster_name>_-mirrormaker2

Reviewer: ```suggestion
|<cluster_name>-mirrormaker2
```

|MirrorMaker 2 pods

> |_<cluster_name>_-bridge

Reviewer: ```suggestion
|<cluster_name>-bridge
```
